index.html

```diff
@@ -36,5 +36,104 @@ <h1>I'm Norman<br> Glad to see you!</h1>
             </ul>
         </div>
     </section>
+    <section class="works-section">
+        <div class="work-card">
+            <div class="work-image">
+                <img src="images/work1.png" alt="work 1">
+            </div>
+            <div class="work-details">
+                <h1>Tonic</h1>
+                <p class="work-subtext">CANOPY<span class="dot"></span><span class="description">Back End Dev</span><span class="dot"></span><span class="description">2015</span> </p>
+                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <p class="project-btn"><a href="">See Project</a></p>
+            </div>
+        </div>
+        <div class="work-card">
+            <div class="work-image">
+                <img src="images/work2.png" alt="work 2">
+            </div>
+            <div class="work-details">
+                <h1>Multi-Post<br> Stories</h1>
+                <p class="work-subtext">CANOPY<span class="dot"></span><span class="description">Back End Dev</span><span class="dot"></span><span class="description">2015</span> </p>
+                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <p class="project-btn"><a href="">See Project</a></p>

```

Comment:
- [ ] I think it would be better to use `<button>` or `<a>` tag here instead of `p` tag. because this is a link 👌. And please don't forget to follow the button styling: Active, Hover, and Pressed.

![2](https://user-images.githubusercontent.com/45231276/176837061-facdfea9-a844-49bc-bed7-93be8ab7b49c.PNG)